README.md

```diff
@@ -195,7 +195,7 @@ Example command:
 
 ### Deployment
 
-You can deploy this project using:
+<!--You can deploy this project using:-->
 
```

Comment:
- [ ] **Please I'M Not able to inline comment in all your README where there is The LICENSE**  but here is the issue : You've done a great job so far on this project, but you still need to improve on something for it to be approved. I noticed that your  `README` file the  M.I.T License is not really working since you provide the wrong link , please include it because using the MIT License in our projects as students at Microverse can provide us  with a number of benefits, including collaboration opportunities, improved skills, portfolio building, career opportunities, and networking., kindly make sure that your M.I.T license is working properly :wink:  I suggest you to have a look on this video on [how to license a project](https://www.youtube.com/watch?v=FLAcvWCxqqM)

![Screenshot from 2023-02-17 16-47-40](https://user-images.githubusercontent.com/88894525/219686662-3086f79e-0e5f-4821-98d0-1547466c79cf.png)